bee-node/src/main.rs

```diff
@@ -19,9 +47,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
         Err(e) => panic!("Failed to create the node config builder: {}", e),
     };
 
-    if let Err(e) = logger_init(config.logger.clone()) {
-        panic!("Failed to initialise the logger: {}", e);
-    }
+    #[cfg(feature = "console")]
+    let console_handle = logger_init();
+
+    #[cfg(not(feature = "console"))]
+    logger_init(config.logger.clone());
```

Comment:
I'm confused, why would we not have logging if we have the console ?

Reply:
The logger is replaced by the `tracing_subscriber`, so if we have already initialised one beforehand, initialising the subscriber panics. The subscriber logs everything it captures to stdout as well as whatever the additional layers direct it to do.

Reply:
Alright, makes sense 👍